src/api_server/swagger/firecracker.yaml

```diff
@@ -699,6 +817,19 @@ definitions:
         description: The amount of milliseconds it takes for the bucket to refill.
         minimum: 0
 
+  Vm:
+    type: object
+    description:
+      Defines the microVM state. It is especially useful in the snapshotting context.
```

Comment:
I guess it is worth saying here that this refers to the running state of the microVM, not to be confused with the microvm state file.

Reply:
Agree, done.